tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java

```diff
@@ -168,24 +167,28 @@ public static void main(String[] args) {
                     Multimap<String, String> valueToSource =
                             TreeMultimap.create(LENGTH_FIRST, Comparator.naturalOrder());
                     for (FormatParameters parameters : FormatParameters.allCldr()) {
-                        Options options;
                         Order order = parameters.getOrder();
+                        //                        Options options; // use this to get the 'none'
+                        // case
+                        //                        if (order == nameOrder) {
+                        //                            options = Options.none;
+                        //                        } else if (order == Order.sorting && nameOrder ==
+                        // myOrder) {
+                        //                            options = Options.sorting;
+                        //                        } else {
+                        //                            continue; // skip otherwise
+                        //                        }
```

Comment:
Looks like you have an automated code formatter that messed up the indentation here...

Beyond that, I guess I should repeat my question from earlier-- does this version of the test somehow test automatic determination of field order based on the formatter and name locales?  Because that was something we were (theoretically) getting out of the old "none" stuff.  Or am I wrong to think the older "none" test cases were actually doing that in a useful way?  If so, does that mean that this set of tests doesn't really exercise that functionality, and that my other unit tests are the only thing checking that, and is that how we want it?

Reply:
I think the 'none' values were doing that in an incomplete way. We do want to test it, but I was focusing here on getting the tests cleaned up.

Reply:
You said above you thought we should find a way to test automatic determination of the name order, but that this isn't the right way, and you've filed a ticket to add something for that.  That all sounds fine to me.